src/build.rs

```diff
@@ -1,11 +1,18 @@
 use crate::cmd::{Command, MountKind, Runnable, SandboxBuilder};
 use crate::prepare::Prepare;
-use crate::{Crate, Toolchain, Workspace, crates::CratePatch};
+use crate::{Crate, Toolchain, Workspace};
 use failure::Error;
 use remove_dir_all::remove_dir_all;
 use std::path::PathBuf;
 use std::vec::Vec;
 
+#[derive(Clone)]
+pub struct CratePatch {
```

Comment:
Currently this type is in the public interface of `prepare::TomlTweaker`.  That being said, I'm not sure that `TomlTweaker` should/needs to be public?  It doesn't appear to be used in crater at all.

The intent behind this was to use it as a common type between rustwide and crater for patches; i'm going to need to parse & store patches on a toolchain in crater to apply them to a build, and I felt it would be cleaner to have a type to do that with.

Reply:
> Currently this type is in the public interface of prepare::TomlTweaker. That being said, I'm not sure that TomlTweaker should/needs to be public? It doesn't appear to be used in crater at all.

Uh, good catch, that was an oversight on my part. You can put `TomlTweaker` private.

> The intent behind this was to use it as a common type between rustwide and crater for patches; i'm going to need to parse & store patches on a toolchain in crater to apply them to a build, and I felt it would be cleaner to have a type to do that with.

Hmm, I's still prefer for this not to be exposed and instead provide the arguments to the builder method directly.